department-of-veterans-affairs/va.gov-cms
# VACMS-1239: Removing field_office from 4 node types.

## Description
See #1239 
This is step 4 of process = remove field_office from Events, News releases, Publications and Stories.
**Step 3 (the front end consumption switch over) was merged into vets-website yesterday. Should be fine to merge in now, but to be safe, we should wait until this afternoon to merge this in**

## Testing done
Dst

## QA steps
As an admin, visit the following paths, and visually verify that field_office does not appear:
 - [x] `/admin/structure/types/manage/event/fields`
 - [x] `/admin/structure/types/manage/news_story/fields`
 - [x] `/admin/structure/types/manage/press_release/fields`
 - [x] `/admin/structure/types/manage/outreach_asset/fields`
